scripts/create_input.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import sys
+
+import predicting_the_past_import as import_data
+
+# user defined characteristics of cyclus simulation
+data_year = 2019
```

Comment:
If this is historically how cyclus has been run, then OK. I think my previous comment about allowing users to choose their basis (atom/mass) still stands. Rather than making a command line argument you could create a new variable here, 
```python
recipe_basis = 'mass' 
```
and call it later on when you write the files. 

In my *opinion* templates should have as many options as possible.

Reply:
This template is not for all of cyclus. This is for a specific simulation.